objects/Fan-MadeAccessories.aa8b38/CleanUpHelper.26cf4b.ttslua

```diff
@@ -207,10 +209,10 @@ function resetCounters()
   end
 
   -- reset doom on agenda
-  local doomcounter = getObjectFromGUID("85c4c6")
-  if doomcounter ~= nil then
-    doomcounter.call("setToZero")
-  end
+  -- local doomcounter = getObjectFromGUID("85c4c6")
+  -- if doomcounter ~= nil then
+  --   doomcounter.call("setToZero")
```

Comment:
If this gets changed to `doomcounter.call("updateVal")`  it would work again

Reply:
Unrelated to this PR, can fix it later.  I probably shouldn't have even commented it out, but it was breaking the testing